app/assets/stylesheets/components/_nav.scss

```
@@ -52,9 +52,9 @@
 }
 
 .top-banner {
-  text-align: center;
+  display: none;
 
   @include at-media('desktop') {
-    text-align: left;
+    display: flex;
   }
 }
```

The `display: flex` here is unfortunate since it creates a dependency between `.top-banner` and `.grid-row`, where in reality we want something more like a [CSS `initial`](https://developer.mozilla.org/en-US/docs/Web/CSS/initial). At first, I thought it would be a limitation of mobile-first styling to only support media queries as "greater than", though in digging further into the USWDS code/documentation, I did find an additional variation of this mixin as `at-media-max` which can work for us:

https://designsystem.digital.gov/design-tokens/spacing-units/#using-spacing-unit-tokens

So instead this could potentially be:

```suggestion
.top-banner {
  @include at-media-max('mobile-lg') {
    display: none;
  }
}
```